pkg/java/prepare/application.go

```diff
@@ -106,7 +113,7 @@ func prepareEffectiveStartscript(scriptPath string) error {
 		return nil
 	}
 
-	for _, altScriptName := range []string{"start.sh", "start", "generated-start"} {
+	for _, altScriptName := range []string{"os-start.sh", "start.sh", "start", "generated-start"} {
```

Comment:
"start" bør ikke være i lista siden vi allerede har sjekket om "start" finnes.

Reply:
Hva skjer her hvis filen start finnes i denne metoden? Slik jeg leser kildekoden så blir det da ikke laget noe script? Stemmer det?

Jeg stemmer for at vi fjerner koden over for blokken. Putter generated-start først og så de andre med os-start.sh/os-start før start.sh/start.